pipeline/splicing/calcVarPriors.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
+
+'''
+calcVarPriors
+
+Parses a tsv file (default built.tsv) containing variant information and for each variant in file 
+calculates either the prior probability of pathogenicity or a prior ENGIMA classification based on variant type and variant location
+'''
+
+import argparse
+import csv
+
+def checkSequence(sequence):
+    '''Checks if a given sequence contains acceptable nucleotides returns True if sequence is comprised entirely of acceptable bases'''
+    acceptableBases = ["A", "C", "T", "G", "N", "R", "Y"]
+    badBases = 0
+    if len(sequence) > 0:
+        for base in sequence:
+            if base not in acceptableBases:
+                badBases += 1
+            if badBases == 0:
+                acceptableSequence = True
+            else:
+                # badBases > 0
```

Comment:
can rem comment

Reply:
one example of a simpler if/else block is:
```
if len(sequence) > 0:
    for base in sequence:
        if base not in acceptableBases:
            return False
    return True
else:
    return False
```
            
no need for the badBases or acceptableSequence variables